src/components/LoginCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, Card, Icon } from 'semantic-ui-react';
+import { NavLink } from 'react-router-dom';
+import '../styles/loginCard.scss';
```

Comment:
I thought we agreed on importing `scss` styles only in the `index.scss`, Please do well to import only in the index file, cheers.

Reply:
This has been fixed now. Thanks